### DISCUSSION TITLE
Reading https traffic from Audible Mobile app

### DISCUSSION THREAD

**Discussion by Mbucari** [mkb79/Audible#98] [General]
Today I tried to see if I could capture all network traffic from the audible mobile app so I could better reverse engineer some of the api. I got it partially working by installing a root certificate authority on my rooted Android device using HTTP Toolkit. I'm able to capture most of the traffic, but the audible app seems to be rejecting the certificate for some of its web requests.

It seems like the audible app is using SSL pinning for some of its web requests, such as sign-in. [I found a tutorial to bypass certificate pinning](https://httptoolkit.tech/blog/frida-certificate-pinning/)  but it looks like I may need a Linux machine to get it working, and I don't have one.

How do you inspect the http traffic? I know you're using an iphone, but I was hoping some of the problems you solved might be applicable to Android as well.

**Comment by mkb79** [comment]
@Mbucari 
Let's talk about this topic by email. It could be that some Amazon software developers may also read here.
